backend/app/usecase/requester/verifier_recaptcha.go

```diff
@@ -0,0 +1,24 @@
+package requester
+
+// VerifierReCaptcha verifies incoming network using ReCaptcha to prevent cyber attacks.
+type VerifierReCaptcha struct {
+	service ReCaptcha
+}
+
+var _ Verifier = (*VerifierReCaptcha)(nil)
```

Comment:
Let's put this before type definition so that it's easier to reader to know the functionality of struct before reading into the details